Do-ItTests/PersistenceTest.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  PersistenceTest.swift
+//  Do-ItTests
+//
+//  Created by user146350 on 1/29/19.
+//  Copyright © 2019 The Swifter Picker-Uppers. All rights reserved.
+//
+
+import XCTest
+@testable import Do_It
+
+class PersistenceTests: XCTestCase {
```

Comment:
This class should be broken up into two separate classes: one to test course persistence, and one to test do-it persistence.